build-linux.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+export JAVA_HOME=/usr/lib/jvm/java-8-oracle
+
+cmake .
+cmake --build . -- -j 8
```

Comment:
why 8?

Reply:
guessed. seems to work reasonable on travis. other suggestions?

---
In reply to: [192359389](https://github.com/JohnLangford/vowpal_wabbit/pull/1492#discussion_r192359389) [](ancestors = 192359389)

Reply:
If you leave it empty i.e. ('-j') it tries to maximize number of parallel compiles.  so if you have more than 8 cores you can get more parallelism in your build.  

---
In reply to: [192386244](https://github.com/JohnLangford/vowpal_wabbit/pull/1492#discussion_r192386244) [](ancestors = 192386244,192359389)